web-client/src/presenter/computeds/Public/todaysOrdersHelper.ts

```diff
@@ -51,15 +48,43 @@ export const todaysOrdersHelper = (
     };
   });
 
-  const hasResults = formattedOrders.length > 0;
+  const tableSort = get(state.tableSort);
 
-  const showLoadMoreButton = formattedOrders.length < totalCount;
+  const sortedFormattedOrders = formattedOrders.sort((orderA, orderB) => {
+    let sortNumber = 0;
+    const compare1 = tableSort?.sortOrder === DESCENDING ? orderB : orderA;
+    const compare2 = tableSort?.sortOrder === DESCENDING ? orderA : orderB;
+
+    if (!tableSort) {
+      sortNumber = compare1.createdAt.localeCompare(compare2.createdAt);
+    } else if (
+      SUPPORTED_SORT_FIELDS_FOR_TODAYS_ORDERS.includes(tableSort.sortField)
```

Comment:
I would recommend updating the name `SUPPORTED_SORT_FIELDS_FOR_TODAYS_ORDERS` or else including docket number in it and updating the implementation here accordingly. We _do_ support sorting by docket number, it just needs to be sorted differently.